### ISSUE TITLE
Add ICD11 as a prefix for Disease

### ISSUE THREAD

**Issue by gaurav** [biolink/biolink-model#1531]
**Is your feature request related to a problem? Please describe.**
Biolink Model supports `ICD9` and `ICD10` as prefixes for Disease:

https://github.com/biolink/biolink-model/blob/a2ec41e872a4acaecc7c30f826f7d6dd4a056b6d/biolink-model.yaml#L8136-L8186

We should also support `ICD11`, which [came out in Nov 2021](https://pmc.ncbi.nlm.nih.gov/articles/PMC8577172/). NodeNorm would like to add support for ICD11 mappings, but this would require `ICD11` becoming an allowed prefix in the Biolink Model.

**What working group (or team) did this request originate from?**
NodeNorm (Translator SRI)

**Note:** This is relevant for members of NCATS Translator.

**Describe the solution you'd like**
Add `ICD11` as a prefix for Disease (presumably just above `ICD10`).

**Additional information to support this request** (optional)
N/A

**Tag relevant members for discussion**
@sierra-moxon

**Comment by gaurav** [comment]
Note that other Monarch Initiative projects use the prefix `icd11.foundation:` and define it as either `http://id.who.int/icd/entity/` [as in monarch-initiative/icd11](https://github.com/monarch-initiative/icd11/blob/40383e8443f9126a1ef3b99328a27a2dffc2faa3/config/icd11.sssom-metadata.yml#L3) or `http://purl.obolibrary.org/obo/mondo/sources/icd11foundation/` [as in MONDO](https://github.com/monarch-initiative/mondo/blob/703f804c75ea08d87eb9621ee1337bca00bd91f9/src/utils/mk-skos.pl#L80-L82).

**Comment by sierra-moxon** [comment]
bioregistry defines `icd11` as the prefix/namespace for this.  It also defines an alternate prefix of `ICD-11`.   There is some desire from Monarch folks to get `icd11.foundation` into bioregistry but it is not yet clear if this will be a prefix alias for the existing `icd11` prefix or a new prefix (this resource has a lot of linearizations).  

@gaurav - which resources are you consuming with this set of cross references?  Primarily MONDO?

**Comment by gaurav** [comment]
Yup, primarily MONDO! I'm happy to translator `icd11.foundation` to `icd11` during ingest into NodeNorm if that would preserve the IRI.

**Comment by colleenXu** [comment]
@sierra-moxon @kevinschaper 

I saw the release candidate and there's an inconsistency in the format of "ICD11". At the top (line 80), the format is all-lowercase `icd11`. In the Disease prefixes (line 8215?), the format is all upper-case `ICD11`. 

Note: if the desired format is `icd11`, I find it odd that the others have all-caps format `ICD9` and ICD10`.

**Comment by sierra-moxon** [comment]
Hi @colleenXu - yes, I agree, it is inconsistent.  We're in the middle of trying to reconcile all of this with bioregistry, bioportal, and linkml/prefixmaps.  I hope by the time this all gets done in those repos, that we'll have a capitalized version of the prefix to match the others, but I wanted to get a "stop-gap" prefix in place in the meantime as this kind of work often takes awhile.

**Comment by colleenXu** [comment]
So in the meantime...should the release candidate be consistent in all lower-case or all-caps?
